src/dialog_launcher.ts

```diff
@@ -111,10 +111,15 @@ export class Launcher extends search.Search {
 
             // Sort the list of matched selections
             this.selections.sort((a, b) => {
-                const a_name = a instanceof window.ShellWindow ? a.name(ext) : a[1].name();
-                const b_name = b instanceof window.ShellWindow ? b.name(ext) : b[1].name();
+                const a_name = a instanceof window.ShellWindow ? a.name(ext).toLowerCase() : a[1].name().toLowerCase();
+                const b_name = b instanceof window.ShellWindow ? b.name(ext).toLowerCase() : b[1].name().toLowerCase();
 
-                return a_name.toLowerCase() > b_name.toLowerCase() ? 1 : 0;
+                const pattern_lower = pattern.toLowerCase()
+
+                const a_includes = a_name.includes(pattern_lower);
```

Comment:
@mmstick would `indexOf` be better here? We could also sort by "how close to start" the position is. I think it's probably not worth doing unless folks still feel the result order needs improving.

Reply:
Ultimately, I've planned to switch to using the levenshtein distance algorithm to compare and weight results.